winui/Shadow/Customization.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: Customization in WinUI Shadow control | Syncfusion
+description: Learn here all about the customization features such as color, opacity, position, and blur radius in Syncfusion WinUI Shadow control.
```

Comment:
description: Learn all about the customization features such as color, opacity, position, and blur radius in Syncfusion's WinUI Shadow control here.

Reply:
Changed the description as mentioned above.